backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/pporder/api/impl/AbstractPPOrderReceiptHUProducer.java

```diff
@@ -310,7 +310,7 @@ public final void createReceiptCandidatesFromPlanningHU(@NonNull final I_M_HU pl
 			}
 
 			//
-			updateReceivedHUs(huContext, ImmutableSet.of(planningHU));
+			updateReceivedHUs(huContext, ImmutableSet.of(planningHU), null);  // TODO tbp: not sure if this should be null
```

Comment:
consider removing createReceiptCandidatesFromPlanningHU() at all.
in some branches, I've already removed.
so u don't have to bother fixing this TODO

Reply:
removed the method from interface and implementation.

Reply:
done